etc/rc.local.linux

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -e
+#
+# /etc/rc.local
+#
+# This script is executed at the end of each multiuser runlevel.
+# This means that once krun has rebooted the machine, you will not
+# see a login prompt until all benchmarks have finished.
+#
+# Verify this script by running:
+#    sudo service rc.local start
+#
+
+# Re-direct STDOUT and STDERR to a log file.
+exec 2>/tmp/rc.local.log
```

Comment:
I think we are getting close to being able to merge.

Perhaps `/tmp` isn't the best place to put the log file, as many systems clear `/tmp` on boot. How about `/var/log`?